swagger.yaml

```diff
@@ -2879,6 +2887,10 @@ definitions:
               format: uuid
             hostname:
               type: string
+      ntp_source:
+        type: string
```

Comment:
Is there a validation on NTP source? for example empty string is valid by this definition. 
A more generic question is how the user know that he configured the wrong NTP source?

Reply:
A host would become insufficient due to host validator `IsNTPSynced` failing.

Reply:
i know that it will fail but will the user understand what is the issue or do we have to connect and take a look at the logs?

Reply:
It's a part of the host validator mechanism. A message would be presented as described on https://github.com/openshift/assisted-service/pull/675/files#diff-d7db3d1eb3c67167cb6aaffa4ed66cd1dcd7d2097d63648ecd03410a36f1283eR534

Reply:
Maybe i'm not clear, if i will write ntp source "blabla" it will probably don't work, is there a way to know if the npt source is not valid?

Reply:
Ok now I understood.
I will add host/IP address validation.

```bash
[core@test-infra-cluster-assisted-installer-master-1 ~]$ chronyc add server blabla
Invalid host/IP address
```